toolhub/templates/home.jinja

```diff
@@ -1,5 +1,61 @@
 {% extends "layouts/base.jinja" %}
+{% from "partials/badges.jinja" import taxonomy_list, availability_badge %}
 
 {% block content %}
-    Home is where the heart is.
+    {% if user.is_authenticated %}
+        <div class="row">
+            <div class="col-sm-6 col-md-4 mb-4">
+                <h2>Your Tools</h2>
+                {% for tool in user.tools.all() %}
```

Comment:
Let's limit it to a certain amount, I had added a settings `DEFAULT_PAGINATE_BY` maybe a division of that or a new env var?
When there are more than the limit link to `tools/owned/` at bottom